src/main/java/core/mate/academy/service/MachineService.java

```diff
@@ -1,12 +1,13 @@
 package core.mate.academy.service;
 
+import core.mate.academy.model.Machine;
+
 import java.util.List;
 
 /**
  * Parameterize this service and add its implementation in a separate class.
  */
-public interface MachineService {
-
+public interface MachineService<M extends Machine> {
```

Comment:
generic types are `T` - commonly accepted
```suggestion
public interface MachineService<T extends Machine> {
```